scheduler/scheduler_sync.daemon

```diff
@@ -9,64 +9,77 @@
 # sync's via Maxwell to all remote sites, utilizing the autossh link between Maxwell and all site
 # computers.
 #
-# This script should be run via a systemd daemon with the following service file placed in 
-# /usr/lib/systemd/system/:
-#   scheduler_sync@.service:
-#       [Unit]
-#       Description=Scheduler synchronization daemon for %i
-#       
-#       [Service]
-#       User=mrcopy
-#       ExecStart=/home/mrcopy/borealis/scheduler/scheduler_sync.daemon %i
-#       Restart=always
-#       
-#       [Install]
-#       WantedBy=multi-user.target
+# This script should be run via a systemd daemon with the following service file:
+# /usr/lib/systemd/system/scheduler_sync@.service
+#    [Unit]
+#    Description=Scheduler synchronization daemon for %i
+#
+#    [Service]
+#    User=mrcopy
+#    ExecStart=/home/mrcopy/borealis/scheduler/scheduler_sync.daemon %i
+#    RuntimeMaxSec=3600s
+#    Restart=always
+#
+#    [Install]
+#    WantedBy=multi-user.target
 #
 # The site to sync to can be specified by calling the daemon with the following syntax:
-#       systemctl [command] scheduler_sync@[RADAR_ID]
+#       systemctl [command] scheduler_sync@[RADAR_NAME]
 #
 # Dependencies:
 #   - inotifywait (installed by zypper in inotify-tools)
-#   - Maxwell is configured via autossh to each site Borealis computer with ports that correspond 
-#   to the SITE_NUM association
-#   - ssh link between SuperDARN-CSSDP -> Maxwell (no password prompt for rsync)
+#   - ssh link between computer and remote radar computers (no password prompt for rsync)
+#
+# Requires:
+#   - file ${HOME}/.scheduler with read permissions
+#       - file must declare an array RADAR_PORTS like
+#           declare -A RADAR_PORTS=(["SAS_BORE"]=xxxxx ["SAS_MAIN"]=yyyyy)
+#         where xxxxx and yyyyy are ports for an ssh connection to the computer.
+#       - file must also define SCHEDULER_DEST as SCHEDULER=username@host
+#   - SLACK_WEBHOOK_[RADAR_ID] defined in ${HOME}/.profile, where RADAR_ID is SAS, PGR, etc.
```

Comment:
It is good to have this documentation here, but the ReadTheDocs should also be updated to reflect scheduler changes